e2e/tests/components/toolbar-search/toolbar-search.po.spec.ts

```diff
@@ -23,4 +23,14 @@ export class ToolbarSearchPage {
 
     // button to set always expanded state
     alwaysExpandedBtn = element(by.id('always-expanded-btn'));
+
+    async clickOnDropdown() {
```

Comment:
Would `openDropdown` be a better name for this? `clickOnDropdown` might imply just clicking somewhere within the open dropdown.